### ISSUE TITLE
Panic messages don't appear in logcat

### ISSUE THREAD

**Issue by jdm** [servo/servo#21783]
This means that assertions, unwrap, etc. don't leave any trace. The log shows a sigill, and I can only figure out what's happening if I run the app under gdb. I wonder if we could add a call to `__android_log_write` in the global panic hook to make this work.

**Comment by SimonSapin** [comment]
This API is unstable and undocumented, but it is public in `std::io`:

```rust
pub fn set_panic(sink: Option<Box<dyn Write + Send>>) -> Option<Box<dyn Write + Send>>
```

It is similar to `set_print` mentioned in https://github.com/servo/servo/issues/21637#issuecomment-419623081.

**Comment by jdm** [comment]
Yes, I have a patch that adapts Servo's existing panic hook to give me logcat output.
